src/System.Device.Gpio/System/Device/Pwm/Channels/UnixPwmChannel.Linux.cs

```diff
@@ -90,6 +89,10 @@ private static int GetPeriodInNanoSeconds(int frequency)
         /// <param name="frequency">The frequency in hertz to set.</param>
         private void SetFrequency(int frequency)
         {
+            if (frequency < 0)
```

Comment:
I think we should throw also for 0, unless that means "disabled"

Reply:
by default in Linux the value is 0 so I don't want to throw in case of 0.